### DISCUSSION TITLE
Einige Kleinigkeiten

### DISCUSSION THREAD

**Discussion by Linebeck0** [rjcncpt/StarCitizen-Deutsch-INI#386] [Übersetzungsvorschläge]
Hallo,
folgende Sachen sind mir in den letzten Tagen aufgefallen, über eine Korrektur würde ich mich freuen.

Die Aufzüge, z.B. in der Reclaimer werden mit "Geh runter" bzw. "Gehe nach oben" übersetzt.
Da es normale aufzüge sind, würde hier ein simples "Herunterfahren" bzw. "Hochfahren" bzw. "auf"/"ab" genügen.

Die Mission "Money from Nothing" ist vollständig kleingeschrieben, wenigstens Money darf bitte ein großes "M" bekommen ;-).

Die Übersetzung für die Schiffswarnung wenn man schaden hat ist ... schwierig. ("WARNUNG: Großes Drehmoment-Ungleichgewicht")
Hier würde ich mir eine Übersetzung wie "Schubstörung erkannt" oder "Triebwerksschaden erkannt" wünschen. Alternativ, wenn man es kürzer machen möchte wäre auch "Triebwerksfehler" genügen ;-)

Des weiteren sind einige Erze als Roh-Erz übersetzt und andere nur als Erz (Kontext: Mining). Hier wäre entweder eine erklärung was ein Roh-Erz ist ganz nett oder einfach alles nur Erz zu benennen.

Viele Grüße

**Comment by Linebeck0** [comment]
![image](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/76116052/1c1234e8-e78c-40dd-996e-e39bb5d49304)
2 weitere Kleinigkeiten gefunden.
Bekommt man ggf. über folgende Regex-Prüfung die Satzanfänge sinnvoll groß?

Ist Orision wirklich eine schwimmende und keine fliegende/schwebende Stadt?
=.*[:punct:]\s+[a-z]

**Reply by rjcncpt** [reply]
Check

**Reply by rjcncpt** [reply]
> Bekommt man ggf. über folgende Regex-Prüfung die Satzanfänge sinnvoll groß?

Nicht wirklich. Es gibt unzählige Beispiele, wie Texte zusammen gestückelt werden. Das hat kein ersichtliches Muster. Da muss man leider aufpassen.

**Comment by greluc** [comment]
Moin,
habe dafür die Issues #394 und #395 aufgemacht. Die Arbeit kann dann dort verfolgt werden.

**Comment by rjcncpt** [comment]
> Die Aufzüge, z.B. in der Reclaimer werden mit "Geh runter" bzw. "Gehe nach oben" übersetzt.

Die einzigen Stellen in der ini Datei sind diese:
```
innerthought_levski_interior_callascend=Gehe nach oben
innerthought_levski_interior_calldescend=Geh runter
```
Wie man sieht, hat das gar keinen bezug zur Reclaimer. Man könnte das zu
```
innerthought_levski_interior_callascend=Nach oben
innerthought_levski_interior_calldescend=Nach unten
```
ändern.

**Comment by rjcncpt** [comment]
> Die Mission "Money from Nothing" ist vollständig kleingeschrieben, wenigstens Money darf bitte ein großes "M" bekommen ;-).

Ja die ist in der engl. ini auch so. Ist korrigiert

**Comment by rjcncpt** [comment]
> Die Übersetzung für die Schiffswarnung wenn man schaden hat ist ... schwierig. ("WARNUNG: Großes Drehmoment-Ungleichgewicht")
> Hier würde ich mir eine Übersetzung wie "Schubstörung erkannt" oder "Triebwerksschaden erkannt" wünschen. Alternativ, wenn man es kürzer machen möchte wäre auch "Triebwerksfehler" genügen ;-)

Hm. In der engl. ini steht folgendes:
`dfm_ui_UnexpectedTorqueImbalance=WARNING: Major Torque Imbalance`

Ist das mit zB. _Triebwerksschaden erkannt_ übersetzbar?

Edit: Ist zu `Triebwerksschaden erkannt` geändert

**Comment by rjcncpt** [comment]
> Des weiteren sind einige Erze als Roh-Erz übersetzt und andere nur als Erz (Kontext: Mining). Hier wäre entweder eine erklärung was ein Roh-Erz ist ganz nett oder einfach alles nur Erz zu benennen.

Ich habe es zu (Raw) geändert. Wie in der engl. ini auch. Die Titel sind ja engl. geblieben. Lediglich in der Beschreibung ist die deutsche Bedeutung aufgeführt

**Comment by Linebeck0** [comment]
Vielen Dank für eure Anpassungen, ich hab leider die nächsten Kleinigkeiten gefunden, diesmal wieder im Kontext Mining.
Schwierigkeitsgrad "Hart" als "Fest"
Erze teilweise als "Erz", teilweise als Raw - Dann bitte alles gleich ;-)
![image](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/76116052/bc35219d-f699-47e7-a12e-8d1d7d3ed361)

Ist es euch eig. lieber das als Diskusion zu haben oder soll ich euch da jeweils Issues draus machen?
